{
	"id": "CVE-2013-2099",
	"description": "Algorithmic complexity vulnerability in the ssl.match_hostname function in Python 3.2.x, 3.3.x, and earlier, and unspecified versions of python-backports-ssl_match_hostname as used for older Python versions, allows remote attackers to cause a denial of service (CPU consumption) via multiple wildcard characters in the common name in a certificate.",
	"hardware": [],
	"software": [
		{
			"name": "bzr",
			"platform": "debian-12",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "bzr",
			"platform": "debian-11",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "bzr",
			"platform": "debian-10",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "bzr",
			"platform": "debian-9",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-12",
			"version": "< 8.5-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-11",
			"version": "< 8.5-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-10",
			"version": "< 8.5-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-9",
			"version": "< 8.5-1"
		},
		{
			"name": "python-tornado",
			"platform": "debian-12",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-tornado",
			"platform": "debian-11",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-tornado",
			"platform": "debian-10",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-tornado",
			"platform": "debian-9",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-12",
			"version": "< 1.6-2"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-11",
			"version": "< 1.6-2"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-10",
			"version": "< 1.6-2"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-9",
			"version": "< 1.6-2"
		},
		{
			"name": "python2.7",
			"platform": "debian-12",
			"version": "< 2.7.5-5"
		},
		{
			"name": "python2.7",
			"platform": "debian-11",
			"version": "< 2.7.5-5"
		},
		{
			"name": "python2.7",
			"platform": "debian-10",
			"version": "< 2.7.5-5"
		},
		{
			"name": "python2.7",
			"platform": "debian-9",
			"version": "< 2.7.5-5"
		},
		{
			"name": "u1db",
			"platform": "debian-10",
			"version": "< 13.10-1"
		},
		{
			"name": "u1db",
			"platform": "debian-9",
			"version": "< 13.10-1"
		},
		{
			"name": "bzr",
			"platform": "debian-12",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "bzr",
			"platform": "debian-11",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "bzr",
			"platform": "debian-10",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "bzr",
			"platform": "debian-9",
			"version": "< 2.6.0~bzr6574-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-12",
			"version": "< 8.5-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-11",
			"version": "< 8.5-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-10",
			"version": "< 8.5-1"
		},
		{
			"name": "linkchecker",
			"platform": "debian-9",
			"version": "< 8.5-1"
		},
		{
			"name": "python-tornado",
			"platform": "debian-12",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-tornado",
			"platform": "debian-11",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-tornado",
			"platform": "debian-10",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-tornado",
			"platform": "debian-9",
			"version": "< 2.4.1-3"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-12",
			"version": "< 1.6-2"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-11",
			"version": "< 1.6-2"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-10",
			"version": "< 1.6-2"
		},
		{
			"name": "python-urllib3",
			"platform": "debian-9",
			"version": "< 1.6-2"
		},
		{
			"name": "python2.7",
			"platform": "debian-12",
			"version": "< 2.7.5-5"
		},
		{
			"name": "python2.7",
			"platform": "debian-11",
			"version": "< 2.7.5-5"
		},
		{
			"name": "python2.7",
			"platform": "debian-10",
			"version": "< 2.7.5-5"
		},
		{
			"name": "python2.7",
			"platform": "debian-9",
			"version": "< 2.7.5-5"
		},
		{
			"name": "u1db",
			"platform": "debian-10",
			"version": "< 13.10-1"
		},
		{
			"name": "u1db",
			"platform": "debian-9",
			"version": "< 13.10-1"
		}
	],
	"references": [
		"https://bugzilla.redhat.com/show_bug.cgi?id=963260",
		"http://www.ubuntu.com/usn/USN-1984-1",
		"http://secunia.com/advisories/55116",
		"https://access.redhat.com/errata/RHSA-2016:1166",
		"http://www.ubuntu.com/usn/USN-1985-1",
		"http://bugs.python.org/issue17980",
		"http://secunia.com/advisories/55107",
		"http://rhn.redhat.com/errata/RHSA-2014-1690.html",
		"http://www.ubuntu.com/usn/USN-1983-1",
		"http://www.openwall.com/lists/oss-security/2013/05/16/6"
	],
	"severity": null,
	"state": "published",
	"_is_edited": false
}